README.md

```diff
@@ -47,6 +47,12 @@ Files matching this pattern will be included in the manifest.
 
 _Default:_ `"**/*.{js,css,png,gif,ico,jpg,map,xml,txt,svg,swf,eot,ttf,woff,woff2}"`
 
+### ignoreFilePattern
+
+Files matching this pattern will _not_ be gzipped even if they match filePattern.
```

Comment:
Can you update this? This plugin doesn't related to gzipping. Perhaps a copy paste error?